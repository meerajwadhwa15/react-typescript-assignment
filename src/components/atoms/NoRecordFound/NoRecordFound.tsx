import React, { FunctionComponent, ReactNode } from 'react'

export type Props = {
  children: ReactNode
}

const NoRecordFound: FunctionComponent<Props> = ({ children }) => (
  <div>{children}</div>
)

export default NoRecordFound
