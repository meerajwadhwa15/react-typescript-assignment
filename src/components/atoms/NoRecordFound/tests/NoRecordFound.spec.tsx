import React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import NoRecordFound, { Props } from '../NoRecordFound'

const props: Props = {
  children: 'test',
}

describe('NoRecordFound Enzyme testing', () => {
  test('renders without crashing', () => {
    shallow(<NoRecordFound {...props} />)
  })
})

describe('NoRecordFound Snapshot testing', () => {
  test('Matches the snapshot', () => {
    const component = create(<NoRecordFound {...props} />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
