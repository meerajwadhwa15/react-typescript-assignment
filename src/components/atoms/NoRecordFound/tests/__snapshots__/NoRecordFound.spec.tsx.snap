// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoRecordFound Snapshot testing Matches the snapshot 1`] = `
<div>
  test
</div>
`;
